README.rst

```diff
@@ -111,6 +111,11 @@ represents the *next* (i.e. not yet released) version.
 Releases occur approximately every two months, unless there is a pressing need
 to do otherwise (e.g. security & serious bug fixes).
 
+0.8.0 (Upcoming)
+++++++++++++++++
+
+* Upgrade acme to 0.21.x
```

Comment:
Should probably also add "Drop official support for Python 2.6." similar to when 3.3 was dropped: https://github.com/zenhack/simp_le#060

Reply:
I agree. Once this has been added I'm happy.

Reply:
Done !